### PR TITLE
fix(prisma-client-transactions-guide): Make $transaction([]) explicit

### PR DESCRIPTION
**Pull request by janpio**
Right now the document refers to the batch transaction API only as "`$transaction` API". But interactive transactions of course also use `$transaction()`, just with a different parameter (functions instead of array).

Calling it the "`$transaction([])` API" at least lets us differentiate.

This needs to be cleaned up on a higher level though as well: https://github.com/prisma/docs/issues/5123

### CODE REVIEW

File: content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx [hunk 1]
```diff
@@ -79,7 +79,7 @@ Consider using nested writes if:
 
 <Tip>
 
-If you [pre-compute your IDs, you can choose between a nested write or using the `$transaction` API](#scenario-pre-computed-ids-and-the-transaction-api).
+If you [pre-compute your IDs, you can choose between a nested write or using the `$transaction([])` API](#scenario-pre-computed-ids-and-the-transaction-api).
```
Comment (ruheni) [inline content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:82]: Q: Would referring to `$transaction([])` as sequential transaction operations make sense? If so, I would be happy to take care of the updates 🤔

Example usage: 
```suggestion
If you [pre-compute your IDs, you can choose between a nested write or using sequential transaction operations](#scenario-pre-computed-ids-and-the-transaction-api).
```

Comment (janpio) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:82]: A further rewrite of these two pages is definitely justified, potentially including your suggestion. I created https://github.com/prisma/docs/issues/5123 for the follow up work here.

Comment (ruheni) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:82]: I'll add this (rewriting the transactions content page) to the agenda for the Docs grooming session later this week with Jon & Niko.

Comment (janpio) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:82]: Could we still approve and merge this in the meantime?

**Review by ruheni — APPROVED** [review]

File: content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx (hunk 1 above)
Comment (ruheni) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:82]: I've created a new GH issue for further updates to our transaction guides.

**Pull request merged**